quickjs.c

```diff
@@ -10086,9 +10076,7 @@ static double js_strtod(const char *p, int radix, BOOL is_float)
 #define ATOD_TYPE_BIG_INT     (1 << 7)
 #define ATOD_TYPE_BIG_FLOAT   (2 << 7)
 #define ATOD_TYPE_BIG_DECIMAL (3 << 7)
-/* assume bigint mode: floats are parsed as integers if no decimal
-   point nor exponent */
-#define ATOD_MODE_BIGINT      (1 << 9) 
+#define ATOD_MODE_UNUSED      (1 << 9) 
```

Comment:
```suggestion
```
Or did you have a reason for keeping it around?

Reply:
Nah.